Tinder.DAL/ApplicationDbContext.cs

```diff
@@ -0,0 +1,36 @@
+﻿
+using Microsoft.EntityFrameworkCore;
+using Tinder.DAL.Entities;
+
+namespace Tinder.DAL;
+public class ApplicationDbContext : DbContext
+{
+    public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : 
+        base(options)
+    {

```

Comment:
You can enable automatic migrations here using "Database" field of DbContext class provided by EF Core